addons/stock/models/stock_move.py

```diff
@@ -444,8 +444,7 @@ def key_virtual_available(move, incoming=False):
                     move.forecast_availability = move.product_uom._compute_quantity(
                         move.reserved_availability, move.product_id.uom_id, rounding_method='HALF-UP')
                 elif move.state == 'draft':
-                    # for move _consuming_picking_types and in draft -> the forecast_availability > 0 if in stock
-                    move.forecast_availability = virtual_available_dict[key_virtual_available(move)][move.product_id.id] - move.product_qty
+                    move.forecast_availability = virtual_available_dict[key_virtual_available(move)][move.product_id.id]
```

Comment:
We can discuss it but for me we should change the view instead to show a green chart if forecast = 0